### DISCUSSION TITLE
so-status reports missing after upgrade. How to remove deprecated containers?

### DISCUSSION THREAD

**Discussion by mciantar** [Security-Onion-Solutions/securityonion#8976] [Unsupported Versions] [answered]
Hi,

I upgraded so from 2.3.110 to 2.3.180 and since right after version 2.3.110, the hive was dropped in favour of cases.

My issue is that after upgrade, so-status is still checking the status of so-cortex, so-thehive and so-thehive-es. How can I stop so-status from checking the status of these containers since it is correct for them to be missing?

Thanks!

**Comment by darth-drew** [comment, marked as answer]
Check out https://github.com/Security-Onion-Solutions/securityonion/discussions/8155. It helped us.

**Reply by darth-drew** [reply]
Since the hive states were likely removed during your upgrade, you shouldnt have to worry about those containers popping back up in opt/so/conf/so-status/so-status.conf. Might want to double check they are gone though

**Reply by mciantar** [reply]
Hi @darth-drew ,

That was it. I just removed those containers from the so-status.conf, and Grid now shows as OK.

Thanks a lot for the pointer.
